src/main/webapp/fellowship/styles.css

```diff
@@ -0,0 +1,11 @@
+.margin-left-0{
+    margin-left: 0 !important;
+}
+.min-h-100vh{
```

Comment:
```suggestion

.min-h-100vh{
```

Reply:
On it

Reply:
Resolved with commit https://github.com/sef-global/sef-site/pull/748/commits/f2d6f67a13c6bbc62464767011b16e79e033fa6b